index.html

```diff
@@ -5,15 +5,79 @@
 	<title>Home</title>
 	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
 	<link rel="stylesheet" href="css/index.css">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
 	<!--[if IE]>
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
-
+<div class="pos-f-t">
+		<div class="collapse" id="navbarToggleExternalContent">
+		  <div class="bg-dark p-4">
+			<h5 class="text-white h4">Collapsed content</h5>
+			<span class="text-muted">Toggleable via the navbar brand.</span>
+		  </div>
+		</div>
+		<nav class="navbar navbar-dark bg-dark">
+		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		  </button>
+		</nav>
+	  </div>
 <body>
 
-	<h1>You got this!  Good luck.</h1>
-
+	<div class="jumbo-home">
+		<div class="container">
+			<h1>Integrity,
+				<br>Excellence, 
+				<br>Progress.</h1>
+		</div>
+	</div>
+
+	<div class="container">
+		<div class="row">
+			<div class="col-md-6">
+				text in here
+			</div>
+			<div class="col-md-12">
+				<img src= 'img/home/home-jumbotron.png' alt='home-jumbotron'class='jumbo-home'>
+			</div>
+		</div>
+	</div>
+
+	<div class="font">
+			<br>Home Services Contact
+	</div>
+
+	<div class="font" class="cool" >
```

Comment:
If you want to add two class names, you can do it inside one class with a space between.  for example here you could have <div class="font cool">